src/org/usfirst/frc/team2335/robot/subsystems/Shooter.java

```diff
@@ -0,0 +1,83 @@
+package org.usfirst.frc.team2335.robot.subsystems;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.TalonSRX;
+import edu.wpi.first.wpilibj.Victor;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+
+public class Shooter extends Subsystem 
+{
+	//Motors for the shooter
+	private Victor feederMotor;
+	private TalonSRX shooterMotor;
+	private TalonSRX intakeMotor;
+	
+	public double motorSpeed = -0.68;
```

Comment:
I think so; every good comment helps, and has the potential to help teach new team members for years (or even weirder to think about, a decade or more!) down the line.